ctapipe/tools/extract_charge_resolution.py

```diff
@@ -64,7 +69,7 @@ def setup(self):
 
         self.eventsource = SimTelEventSource(**kwargs)
 
-        extractor = ChargeExtractorFactory.produce(**kwargs)
+        extractor = ChargeExtractor.from_name(self.extractor_product, **kwargs)
```

Comment:
This won't work anymore, right?

Reply:
So we are missing a test for this tool. Well spottet! Thanks .. I will try to add one .. 
Or does the missing of the test imply that this tool is not really needed and should be removed?

Reply:
I believe I improve the testing for this tool in #827